src/test/java/YevheniiBondarenko/Tests/Tests/DJITest1E2EbuyingCamera.java

```diff
@@ -0,0 +1,26 @@
+package YevheniiBondarenko.Tests.Tests;
+
+import BasePageAndTestInit.TestInit;
+import YevheniiBondarenko.Tests.Pages.DJIWebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class DJITest1E2EbuyingCamera extends TestInit {
+
+    @Test
+
+    public void djiTestOne(){
+        DJIWebElement djiWebElement = new DJIWebElement(driver);
```

Comment:
please add empty line between 13 and 14 line

Reply:
+